code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -100,21 +100,17 @@
 	for(var/i in 1 to GLOB.configuration.gamemode.traitor_objectives_amount)
 		forge_single_human_objective()
 
-	// Die a glorious death objective.
+	// Are they required to stay alive for their objectives?
+	var/martyr_compatibility = FALSE
```

Comment:
Nitpick but I think this flows better with setting this to `TRUE` initially, then `FALSE` on Ln 109. And on 113, checking `&& martyr_compatibility`.